src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -303,6 +307,27 @@ const OrganizationStream = createReactClass({
     return `/organizations/${params.orgId}/issues/`;
   },
 
+  onSavedSearchChange() {
+    if (!this.state.savedSearchList) {
+      return;
+    }
+
+    let {searchId} = this.props.params;
+    let match = this.state.savedSearchList.find(search => search.id === searchId);
+    if (match) {
+      let projects = [];
+      if (match.projectId) {
+        projects = [parseInt(match.projectId, 10)];
+      }
+
+      // Will trigger a transition if the projects changed
+      GlobalSelectionStore.updateProjects(projects);
```

Comment:
Is there a reason we don't want to use the globalSelection action creator here?

Reply:
No there isn't I'll fix that up before merging.